flow/layers/layer.cc

```diff
@@ -96,7 +111,19 @@ Layer::AutoSaveLayer Layer::AutoSaveLayer::Create(
   return Layer::AutoSaveLayer(paint_context, layer_rec, save_mode);
 }
 
+Layer::AutoSaveLayer Layer::AutoSaveLayer::Create(
+    DisplayListBuilder* builder,
+    const SkRect& bounds,
+    const DlPaint* paint,
+    const DlImageFilter* backdrop) {
+  return Layer::AutoSaveLayer(builder, bounds, paint, backdrop);
+}
+
 Layer::AutoSaveLayer::~AutoSaveLayer() {
+  if (type_ == Type::kDisplayList) {
```

Comment:
Can't we do checkerboarding? Ah, I see an issue. Another comment below would fix this...